pkgs/development/python-modules/blinker/default.nix

```diff
@@ -9,6 +9,9 @@ buildPythonPackage rec {
     sha256 = "1dpq0vb01p36jjwbhhd08ylvrnyvcc82yxx3mwjx6awrycjyw6j7";
   };
 
+  propagatedBuildInputs = [ nose pytestCheckHook ];
```

Comment:
```suggestion
  checkInputs = [ nose pytestCheckHook ];
```

🤦🏼

Reply:
Yes, would be better to place those two there :wink: